src/replicator.js

```diff
@@ -84,7 +87,9 @@ class Replicator {
 
   async _init (opts) {
     this._pubsub = new Pubsub(this.ipfs, (await this.ipfs.id()).id)
-    this._orbitdb = await OrbitDB.createInstance(this.ipfs, { directory: opts.orbitPath })
+    // Identity not used, passes ref to 3ID orbit identity provider
+    const identity = await Identities.createIdentity({ id: 'nullid' })
+    this._orbitdb = await OrbitDB.createInstance(this.ipfs, { directory: opts.orbitPath, identity })
```

Comment:
Hm, shouldn't `createInstance` create an identity automatically? I don't get why this is needed.